### ISSUE TITLE
F-Droid 

### ISSUE THREAD

**Issue by kristaller486** [Vali-98/ChatterUI#49]
It will be It would be a convenient way to get app updates.

From the [website](https://f-droid.org/en/):
F-Droid is an installable catalogue of FOSS (Free and Open Source Software) applications for the Android platform. The client makes it easy to browse, install, and keep track of updates on your device.

**Comment by Vali-98** [comment]
I was considering this for a while, but the app is still in a pretty rough state to provide stable updates via F-Droid. IMO, I need to release a proper solidified 1.0.0 version before this can be done. That said, its still in the plans.

**Comment by Vali-98** [comment]
@cosmojg Thanks for the encouraging words. I would very much like to do it myself as a learning experience. At the moment, I do have a few priority goals for the project which I feel should be done before a F-Droid and potential google playstore release, such as properly migrating Sampler presets from an older file-based system to the the sqlite database and a proper styling rework.

**Comment by CyborgArmy83** [comment]
Another vote for F-droid! It would make staying up to date much easier! It will also ensure that people don't report bugs that might already be resolved by a new build.

**Comment by yazhamuthan-aruchamy** [comment]
Yes, you guys are right can you publish this on the FDroid, Now a days company like snapdragon and mediatek doing the AI realed SOC for mobile is great, So we can able to consider this project an huge success in the upcoming days also, I would like to consider the privacy for my chats in that case this is really an huge One. @Vali-98 hands off 🫡 to your works.

**Comment by Vali-98** [comment]
This is still very much in the cards. I think theres still the API rework, preset reworks and a lot more touchups to do before I'm comfortable with publishing to F-Droid.

There is also the possibility of updating using expo-updates:
https://docs.expo.dev/versions/latest/sdk/updates/

This would bypass the need for a store implementation entirely aside updating internal libraries while providing fast over-the-air updates.

**Comment by lrq3000** [comment]
There is an alternative that is easier to get included in: IzzyOnDroid. Just open a ticket to ask to be included and usually you will be soon enough: https://apt.izzysoft.de/fdroid/index/info

There are less people who know about IzzyOnDroid than F-Droid, but it still allows for easy updates, and you can add a badge on your repo on github to directly link to the app. Personally I use Droid-ify, which includes a lot of different repos including F-Droid and IzzyOnDroid, and I think this kind of multi-repos stores apps is only going to get more traction with the anticompetitive commissions in USA and Europe requiring more and more from Google and Apple to open their ecosystems to competitor stores.

**Comment by bi4key** [comment]
> It will be It would be a convenient way to get app updates.
> 
> From the [website](https://f-droid.org/en/): F-Droid is an installable catalogue of FOSS (Free and Open Source Software) applications for the Android platform. The client makes it easy to browse, install, and keep track of updates on your device.

For now you can install this app:
https://github.com/ImranR98/Obtainium

And add link to App what you want follow:
https://github.com/Vali-98/ChatterUI/releases

Then Obtainium app can follow all you added repo and when new version appear you will have option to simple update app by one click

**Comment by ThreeDeeJay** [comment]
Alternatively, at least in the meantime, we can just use [Obtainium](https://github.com/ImranR98/Obtainium) [[F-Droid](https://f-droid.org/packages/dev.imranr.obtainium.fdroid/)]

**Comment by Slug-Cat** [comment]
I think people on F-Droid are a lot more understanding of bugginess than play store and you may reach a greater number of people willing to offer constructive feedback and perhaps even submit pull requests. It took a word of mouth recommendation for me to find this app, for example, because it isn't really published anywhere.

**Comment by lrq3000** [comment]
It's really an awesome app working very well as it is now, I also think it is beyond ready to be on F-Droid.

**Comment by Vali-98** [comment]
Discussion on this will be moved to #550
